### ISSUE TITLE
Menuconfig for ESP-IDF does not start after update

### ISSUE THREAD

**Issue by SinglWolf** [kisvegabor/lvgl#2637]
VSCode+PlatformIO. Windows 10.
```
CMake Error at components/lvgl/CMakeLists.txt:1 (project):

    project command is not scriptable
```
If commenting out the first 2 lines in the  file `lvgl\CMakeLists.txt`, everything works.
```
# project(lvgl
#   HOMEPAGE_URL https://github.com/lvgl/lvgl/)
```

**Comment by embeddedt** [comment]
Thanks for reporting this; we just discovered it a couple hours ago while working on adding ESP32 CI tests. It will be fixed shortly (if it's not already fixed by #2640).

**Comment by embeddedt** [comment]
It seems to be fixed now on the CI; can you confirm it's fixed for you as well?
